### PR TITLE
misc cleanup

### PR DESCRIPTION
**Pull request by horwitz**

### CODE REVIEW

File: src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt [hunk 1]
```diff
@@ -43,12 +39,11 @@ class QueryEngine(private val dbProvider: DBProvider) {
         ): List<TableRecord> {
             val timer = Stopwatch.createStarted()
             val rs = statement(dbProvider, queryTemplate, binds).executeQuery()
-            timer.reset()
             logger.info("exec query: $timer")
```
Comment (horwitz) [inline src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt:42]: I think the current version will have the first `logger.info` always print a zeroed (reset) timer...

File: src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt [hunk 2]
```diff
@@ -43,12 +39,11 @@ class QueryEngine(private val dbProvider: DBProvider) {
...
+            timer.reset()
             timer.start()
             val records: List<TableRecord> = extractor.invoke(rs)
             logger.info("extract results: $timer")
-            timer.stop()
```
Comment (horwitz) [inline src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt:49]: any reason to stop it? `timer` is about to go out scope anyway

File: src/main/kotlin/org/aarbizu/baseballDatabankFrontend/records/PaginatedRecords.kt [hunk 3]
```diff
@@ -17,29 +19,23 @@ class PaginatedRecords(
     private val pageSize: Int = 10
 ) {
     private val size = records.size
-    private var from: Int = 0
-    private var to: Int = if (size < pageSize) size else from + pageSize
+    private var from = 0
+    private var to = min(from + pageSize, size)
```
Comment (horwitz) [inline src/main/kotlin/org/aarbizu/baseballDatabankFrontend/records/PaginatedRecords.kt:23]: replace first `pageSize` with `from + pageSize` and then simplify(?) to `min` logic

File: src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt (hunk 2 above)
Comment (aarbizu) [reply src/main/kotlin/org/aarbizu/baseballDatabankFrontend/db/QueryEngine.kt:49]: excessive pedantry :P

**Review by aarbizu — APPROVED** [review]
lgtm, like the min() idiom for the paginated records

**Pull request merged**